AsyncDisplayKit/TextKit/ASTextKitContext.mm

```diff
@@ -36,8 +35,8 @@ - (instancetype)initWithAttributedString:(NSAttributedString *)attributedString
 {
   if (self = [super init]) {
     // Concurrently initialising TextKit components crashes (rdar://18448377) so we use a global lock.
-    static std::mutex __static_mutex;
-    std::lock_guard<std::mutex> l(__static_mutex);
+    static ASDN::Mutex __staticMutex;
+    ASDN::MutexLocker l(__staticMutex);
```

Comment:
Not a blocking issue but in the future we may want to switch to [`ASDN::StaticMutex`](https://github.com/facebook/AsyncDisplayKit/blob/master/AsyncDisplayKit/Details/ASThread.h#L167). I believe [ASLayoutable](https://github.com/facebook/AsyncDisplayKit/blob/master/AsyncDisplayKit/Layout/ASLayoutable.mm#L51) is the only place it is used as of now, although this thing has existed since the legendary "Initial commit" :P